package.json

```diff
@@ -95,6 +96,7 @@
     "clipboard": "1.5.5",
     "commander": "2.8.1",
     "css-loader": "0.17.0",
+    "csv": "0.4.6",
```

Comment:
Can you get away with just using the [`csv-parse`](https://www.npmjs.com/package/csv-parse) package, which is one part of the larger `csv` package?